tests/parsers/test_vasp.py

```diff
@@ -11,7 +11,7 @@ class TestVASPParser(unittest.TestCase):
     def get_parser(self,name):
         '''Get a VaspParser for a certain test'''
         unpack_example(os.path.join('examples', 'vasp', name+'.tar.gz'))
-        return VaspParser(name)
+        return VaspParser(name, files=None)
```

Comment:
Why set it directly? It actually re-scopes the test from "default value of `files`" to "`files=None`", which introduces a blind spot.